### PR TITLE
docs(contributing): Added page with instructions for making a change using the GitHub UI

### PR DESCRIPTION
**Pull request by celanthe**
Added page with instructions for making a change using the GitHub UI

### CODE REVIEW

File: community/contributing/docs/github-changes.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+title:  "Make a Change Using the GitHub UI"
+sidebar:
+  nav: community
+---
+
+{% include toc %}
+
+# Open a pull request on GitHub
+
+1. On the page where you see the issue, select the pencil icon at the top right. You can also scroll to the bottom of the page and select **Edit this page**.
```
Comment (317brian) [inline community/contributing/docs/github-changes.md]: This is on the markdown file correct? When first reading this, I thought you were talking about approaching this from spinnaker.io cause there are buttons like Suggest an edit etc on each documentation page https://s.armory.io/eDu6QX5N

So I think it'd be good to clarify where we're starting. Maybe have step 1 be 

`1. Navigate to the Spinnaker documentation repository[https://github.com/spinnaker/spinnaker.github.io]`

Another suggestion is to tell people the top nav of Spinnaker.io corresponds to folder names to help them find the file they're looking for if they're not coming from one of the edit buttons on spinnaker.io.

**Comment by aimeeu** [comment]
Partial: #1707

File: community/contributing/docs/github-changes.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+1. Make your changes in the GitHub markdown editor.
+1. Below the editor, fill in the **Propose file change** form.
+
+    In the **first field**, give your commit message a title that explains what your pull request is about.
+
+    The Spinnaker repositories use a PR title checker, so the title of your PR must follow a specific format: `<type>(<scope>): <subject>`
```
Comment (317brian) [inline community/contributing/docs/github-changes.md]: Missing a period after subject.

File: community/contributing/docs/github-changes.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+    Make sure you include a space after the colon.
+
+    For example:
+
+        docs(plugins): add documentation for plugin creators
+
+    For more information, see [commit message guidelines](https://www.spinnaker.io/community/contributing/submitting/#commit-message-conventions).
+
+    In the **second field**, provide a clear and detailed description of your PR. Please do not to leave these feilds blank, as it is helpful to reviewers when merging your request to have additional context as to what your pull request is about.
```
Comment (317brian) [inline community/contributing/docs/github-changes.md]: Remove `Please`

File: community/contributing/docs/github-changes.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+1. Select **propose file change**.
+1. Select **create pull request**.
+1. After the **open a pull request** screen appears, fill in the form with the following information:
+      * The **subject** field of the pull request defaults to the commit summary. Please change this to add a brief summary of your PR changes.
+      *  The **body** contains some template text. Delete this template text, or feel free to use it to help draft your own extended commit message detailing your pull request. PR descriptions are the first step to helping reviewers and project maintainers understand why your change was made. Any description is better than leaving this field blank, and will help get your PR merged faster!
+      *  Please leave the **allow edits from maintainers** checkbox selected.
+
+# Address feedback in GitHub
+
+Before a pull request is merged, Spinnaker community members will review it. If you have a specific person in mind that you would like to review your pull request, [tag them in the issue comments](https://github.blog/2011-03-23-mention-somebody-they-re-notified/) using the @ symbol and then their GitHub username.
```
Comment (317brian) [inline community/contributing/docs/github-changes.md:45]: remove `will`. It's not needed.

File: community/contributing/docs/github-changes.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
+# What to do if you're asked to make changes
+
+1. Go to the **files changed** tab in GitHub.
```
Comment (317brian) [inline community/contributing/docs/github-changes.md]: Capitalize F on `files`

File: community/contributing/docs/github-changes.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
+---
+layout: single
+title:  "Make a Change Using the GitHub UI"
```
Comment (brian-armory) [inline community/contributing/docs/github-changes.md:3]: GitHub Desktop client/UI? This is not the web UI that's being discussed in this and might be confusing for people.

Comment (317brian) [inline community/contributing/docs/github-changes.md]: GitHub Web UI, right? Cause there's also a desktop client w/ a UI

File: community/contributing/docs/github-changes.md [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+      *  Please leave the **allow edits from maintainers** checkbox selected.
+
+# Address feedback in GitHub
+
+Before a pull request is merged, Spinnaker community members will review it. If you have a specific person in mind that you would like to review your pull request, [tag them in the issue comments](https://github.blog/2011-03-23-mention-somebody-they-re-notified/) using the @ symbol and then their GitHub username.
+
+# What to do if you're asked to make changes
+
+1. Go to the **files changed** tab in GitHub.
+1. Select the edit (pencil) icon on any files that were changed by the pull request.
```
Comment (317brian) [inline community/contributing/docs/github-changes.md]: I don't see a pencil icon on `Files changed`: https://s.armory.io/eDu6Q7kq

File: community/contributing/docs/github-changes.md (hunk 1 above)
Comment (celanthe) [reply community/contributing/docs/github-changes.md]: Done! I've made this change and it's now a new commit.

File: community/contributing/docs/github-changes.md (hunk 2 above)
Comment (celanthe) [reply community/contributing/docs/github-changes.md]: Done! I've made this change and it's now a new commit.

File: community/contributing/docs/github-changes.md (hunk 3 above)
Comment (celanthe) [reply community/contributing/docs/github-changes.md]: Done! I've made this change and it's now a new commit.

File: community/contributing/docs/github-changes.md (hunk 4 above)
Comment (celanthe) [reply community/contributing/docs/github-changes.md:45]: Done! I've made this change and it's now a new commit.

File: community/contributing/docs/github-changes.md (hunk 5 above)
Comment (celanthe) [reply community/contributing/docs/github-changes.md]: Done!

File: community/contributing/docs/github-changes.md (hunk 6 above)
Comment (celanthe) [reply community/contributing/docs/github-changes.md]: I updated the title and .yml nav to reflect this change

File: community/contributing/docs/github-changes.md (hunk 7 above)
Comment (celanthe) [reply community/contributing/docs/github-changes.md]: Removed this particular segment from the list!

**Review by 317brian — APPROVED** [review]
LGTM after the changes

File: community/contributing/docs/github-changes.md [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+
+Before a pull request is merged, Spinnaker community members will review it. If you have a specific person in mind that you would like to review your pull request, [tag them in the issue comments](https://github.blog/2011-03-23-mention-somebody-they-re-notified/) using the @ symbol and then their GitHub username.
+
+# What to do if you're asked to make changes
+
+1. Go to the **Files changed** tab in GitHub.
+1. Make the requested changes.
+1. Commit the changes.
+
+If you find yourself waiting for a reviewer to approve your changes, please feel free to reach out to them once per week. You can also post a message in the <code>#sig-documentation</code> [Slack Channel](https://join.spinnaker.io/).
```
Comment (317brian) [inline community/contributing/docs/github-changes.md]: Is there a reason this needs to be here when the Need help section is immediately following it?  They both essentially say the same thing.

File: community/contributing/docs/github-changes.md [hunk 9]
```diff
@@ -0,0 +1,52 @@
...
+
+# Need help? Get in touch!
+
+If you run into any issues, don't hesitate to reach out to us. We're here to help. Please post a message in the <code>#sig-documentation</code> [Slack Channel](https://join.spinnaker.io/), and someone will get back to you!
```
Comment (317brian) [inline community/contributing/docs/github-changes.md]: Is there a reason this is the join link instead of the direct link to the sig channel?

**Review by 317brian — CHANGES_REQUESTED** [review]
Good to go pending the last two comments

File: community/contributing/docs/github-changes.md (hunk 8 above)
Comment (celanthe) [reply community/contributing/docs/github-changes.md]: Removed this, thanks! :)

File: community/contributing/docs/github-changes.md (hunk 9 above)
Comment (celanthe) [reply community/contributing/docs/github-changes.md]: Updated this with a direct link to the SIG channel :)

**Review by 317brian — APPROVED** [review]

**Pull request merged**